etl/bulk_plot.py

```diff
@@ -0,0 +1,547 @@
+"""HMS Flow Analysis Module.
+
+This module provides functionality for querying and plotting all HMS
+flow data from FFRD S3 storage, including AMS (Annual Maximum Series)
+peaks and confidence limits.
+"""
+
+from __future__ import annotations
+
+import logging
+import os
+from pathlib import Path
+
+import duckdb
+import geopandas as gpd
+import matplotlib.pyplot as plt
+import pandas as pd
+import s3fs
+from dotenv import load_dotenv
+
+__all__ = ["plot_all_hms_elements"]
+
+_ = load_dotenv()
+
+logger = logging.getLogger(__name__)
```

Comment:
can we also consolidate the logging please?

Reply:
Sure.